lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskServiceImpl.java

```diff
@@ -209,6 +212,10 @@ public Task createTask(Task taskToCreate)
       workbasketService.checkAuthorization(
           task.getWorkbasketSummary().getId(), WorkbasketPermission.APPEND);
 
+      if (CreateTaskPreprocessorManager.isCreateTaskPreprocessorEnabled()) {
```

Comment:
Why did you put it here? I would have expected it even before the opening of the connection to the database...

Reply:
I wasn't sure if the workbasket/summary would definately never be needed in the SPI method....changed it to before opening the connection of the database now